presto-orc/src/main/java/com/facebook/presto/orc/stream/InMapOutputStream.java

```diff
@@ -41,6 +41,6 @@ public StreamDataOutput getStreamDataOutput(int column)
     @Override
     public long getRetainedBytes()
     {
-        return INSTANCE_SIZE + super.getRetainedBytes();
+        return INSTANCE_SIZE;
```

Comment:
Is BooleanOutputStream#byteOutputStream retained bytes included here ?

Reply:
I messed this up, will post a fix.